tests/subsys/suit/cache_sink/src/main.c

```diff
@@ -12,6 +12,10 @@
 #include <suit_dfu_cache_rw.h>
 #include <suit_plat_mem_util.h>
 
+<<<<<<< HEAD
```

Comment:
Fixing the git conflict markers in this file should get this PR through the twister stage

Reply:
done. thanks :)